staging/src/k8s.io/pod-security-admission/webhook/manifests/10-namespace.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: pod-security-webhook
\ No newline at end of file
+  name: pod-security-webhook
+  labels:
+    pod-security.kubernetes.io/enforce: restricted
```

Comment:
Consider adding a comment here that the webhook configuration excludes this namespace by default, and we don't actually expect this label to be enforced.

Reply:
commented